build.mill

```diff
@@ -417,6 +417,7 @@ trait MillBaseTestsModule extends TestModule {
       s"-DMILL_SCALA_2_13_VERSION=${Deps.scalaVersion}",
       s"-DMILL_SCALA_2_12_VERSION=${Deps.workerScalaVersion212}",
       s"-DTEST_SCALA_2_13_VERSION=${Deps.testScala213Version}",
+      s"-DMILL_TEST_SCALA_2_13_VERSION=${Deps.testScala213Version}",
```

Comment:
This is already defined as `TEST_SCALA_2_13_VERSION`.

Reply:
Right, that's a debug leftover